# Generated by Django 3.0.2 on 2020-04-17 15:41

import django.contrib.postgres.indexes
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('geostore', '0036_auto_20200116_0926'),
    ]

    operations = [
        migrations.AddIndex(
            model_name='feature',
            index=django.contrib.postgres.indexes.GinIndex(fields=['properties'], name='properties_gin_index'),
        ),
        migrations.AddIndex(
            model_name='featureextrageom',
            index=models.Index(fields=['identifier'], name='geostore_fe_identif_e05782_idx'),
        ),
        migrations.AddIndex(
            model_name='featureextrageom',
            index=django.contrib.postgres.indexes.GinIndex(fields=['properties'], name='feg_properties_gin_index'),
        ),
        migrations.AddConstraint(
            model_name='featureextrageom',
            constraint=models.CheckConstraint(check=models.Q(geom__isvalid=True), name='feg_geom_is_valid'),
        ),
    ]
